app/modules/wifi.c

```diff
@@ -1158,8 +1158,8 @@ static int wifi_station_listap( lua_State* L )
   {
     return luaL_error( L, "Can't list ap in SOFTAP mode" );
   }
-  struct scan_config scan_cfg;
-  memset(&scan_cfg, 0, sizeof(scan_cfg));
+  // set safe defaults for scan time, all others members are initialized with 0
```

Comment:
I like referencing issues & discussions in source code comments. Hence, I would add `// https://github.com/espressif/ESP8266_NONOS_SDK/issues/103` here, what do you think?

Reply:
Sure, why not.